docs/SOPs/Managed_access/Data_Contributor_Agreement_SOP.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Data Contributor Agreement SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 2
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Data Contributor Agreement SOP
+
+This document explains how a wrangler registers a managed access project and helps fill out the data contributor agreement (DCA). See [here](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/Managed_access/Data_Contributor_Agreement_explained.html) if you are unfamiliar with the DCA.
```

Comment:
add ..
registers a managed access project _in HCA Data Repository Ingest Service_  and helps....